packages/kbn-security-solution-side-nav/README.mdx

```diff
@@ -0,0 +1,50 @@
+## Security Solution Side Nav
```

Comment:
nit: I would consider `packages/security_solution/side_nav` to make it easier to find/organize.